DependencyInjection/Configuration.php

```diff
@@ -243,7 +243,7 @@ private function addServiceSection(ArrayNodeDefinition $node)
                 ->arrayNode('service')
                     ->addDefaultsIfNotSet()
                         ->children()
-                            ->scalarNode('mailer')->defaultValue('fos_user.mailer.default')->end()
+                            ->scalarNode('mailer')->defaultValue('fos_user.mailer.twig_swift')->end()
```

Comment:
cannot be done. This is a BC break for any project not configuring the mailer explicitly (and it changes the expected structure of the template)